dataproc/tests/create_cluster_json_test.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the pytest file for creating a Cloud Dataproc cluster using
+# the Python client library and a json configuration.
+
+import os
+
+import pytest
+from google.cloud import dataproc_v1 as dataproc
+
+from dataproc import create_cluster_json
+
+PROJECT_ID = os.environ['GCLOUD_PROJECT']
+REGION = 'us-central1'
+CLUSTER_NAME = 'my-cluster'
```

Comment:
Can you do:

```python
CLUSTER_NAME = 'test-cluster-{}'.format(uuid.uuid4())
```
?

Reply:
Which means you have to replace the value in the config file too.
It's cumbersome, but this is required for running multiple builds (we have presubmit builds for multiple python versions).

Reply:
I think this will be difficult. This JSON file is being consumed by dataproc_v1beta2.ClusterControllerClient to create the cluster. The cluster name is in the JSON.

Reply:
I think this will be difficult. This JSON file is being consumed by dataproc_v1beta2.ClusterControllerClient to create the cluster. The cluster name is in the JSON.

Reply:
@rajib76
Even if it's not simple, we need to avoid test failures. I can not merge this unless having test which pass multiple simultaneous builds.

One idea is to read the json file, then override the value, then write to a temporary file, then run the test with the temporary file as the input.

Sorry it's cumbersome, but we need to run multiple builds safely. Please understand.

Reply:
@tmatsuo 

I have made the required changes by using jinja2 template. It wil l be able to dynamically generate unique cluster name now

Thanks
Rajib